request_handlers.go

```diff
@@ -0,0 +1,212 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+	"strings"
+	"text/template"
+	"time"
+)
+
+func ShowRoot(w http.ResponseWriter, r *http.Request) {
+	logHandlerIntro(r.Method, r.URL.Path, r.Form)
+	renderPage(w, r, "templates/index.tmpl", "Home")
+}
+
+func ShowRegistration(w http.ResponseWriter, r *http.Request) {
+	logHandlerIntro(r.Method, r.URL.Path, r.Form)
+	renderPage(w, r, "templates/registration.tmpl", "Registration")
+}
+
+func ShowLogin(w http.ResponseWriter, r *http.Request) {
+	logHandlerIntro(r.Method, r.URL.Path, r.Form)
+	renderPage(w, r, "templates/login.tmpl", "Login")
+}
+
+func CreateUser(w http.ResponseWriter, r *http.Request) {
+	r.ParseForm()
+	logHandlerIntro(r.Method, r.URL.Path, r.Form)
+
+	newUser := User{
+		Username: r.Form["username"][0],
+		Password: r.Form["password"][0],
+	}
+
+	row := db.QueryRowx(
```

Comment:
I assume we don't care about security at the moment? (SQL Injection delicious) @GrooveStomp

Reply:
**sqlx**'s APIs protect against SQL injection, see: https://jmoiron.github.io/sqlx/#bindvars
Bindvars for PostgreSQL are in the form `$1`, `$2`, etc.